packages/documentation/src/layouts/SidebarLayout.jsx

```diff
@@ -1,17 +1,17 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import Layout from './Layout';
 import Sidebar from '../components/sidebar/Sidebar';
-import PageFooter from '../components/pageFooter/PageFooter';
```

Comment:
It seems like the `PageFooter` component renders the `Components` and `Documentation` components. Are we losing any valuable navigation functionality by deleting the `PageFooter` component, and ergo the `Components` and `Documentation` components?

Reply:
If you look into the `Components` and `Documentation` components, you'll see they implemented rather roundabout ways of rendering the edit link. And that was all `PageFooter` was really for.

Reply:
That makes sense. Thanks for the explanation.